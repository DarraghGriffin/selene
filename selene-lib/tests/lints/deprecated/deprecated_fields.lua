local _ = this.is_deprecated
