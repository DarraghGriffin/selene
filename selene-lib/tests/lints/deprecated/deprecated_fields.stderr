error[deprecated]: standard library property `this.is_deprecated` is deprecated
  ┌─ deprecated_fields.lua:1:11
  │
1 │ local _ = this.is_deprecated
  │           ^^^^^^^^^^^^^^^^^^
  │
  = This field is deprecated :(

